theme.scss

```diff
@@ -0,0 +1,24 @@
+// theme
+// colours
+$theme-colors: (
+  "primary": #000000,
+  "secondary": #ffffff,
+  "dark": #000000
```

Comment:
Why use this if primary is already black?

Reply:
Bootstrap has a default dark that's very similar to black. Anything using the dark colored wouldnt fit wirh anything else